docs/source/course/index.md

```diff
@@ -0,0 +1,66 @@
+# Learn Kedro with hands-on video
+
+If you prefer to learn by following a video tutorial, you can follow the hands-on video course about Kedro
+
+The course is structured into sections, which are further divided in short videos covering a specific Kedro topics. You'll walk through the [spaceflights tutorial](../tutorial/index.md) to get hands-on with that example and rapidly pick up key Kedro concepts.
+
+## Who is this course for?
+
+This course is for data scientists, data engineers and machine learning engineers. You can be junior, mid-level or senior in your field of work. You're likely to be hands-on with projects, or a decision-maker who regularly makes design and implementation choices about Python data products.
+
+We assume you know these concepts:
+* Python basics (coding on Jupyter and other notebook interfaces)
+* Manipulating data with pandas
+* Visualising insights
+* Command line basics
+
+We don't assume knowledge of software engineering in Python, so the course contains information about reusability principles, how to create a Python package, and how to use version control.
+
+## What you'll learn
+
+In short, you'll learn how create maintainable, reusable data science code.
+
+- [x] What Kedro is: the problems is solves and how it fits in the data science ecosystem
```

Comment:
This needs `myst_enable_checkboxes = True` in `conf.py` https://myst-parser.readthedocs.io/en/latest/configuration.html#extensions

Reply:
Thanks! I must be doing something wrong as I can't see it even with that flag. Also tried to format with html and that was a big fail. Probably just use bullets rather than waste any more time 🤦‍♀️